etc/org.opencastproject.adminui.cfg

```diff
@@ -53,6 +53,27 @@
 # Default: */silence
 #smil.silence.flavor=*/silence
 
+# Thumbnail preview mode
+#
+# The video editor can display preview images of the thumbnail that will be generated by the workflows.
+# Those previews are initially generated by the workflow that generates all previews required by the video editor
+# and are later updated ad hoc.
+# The ad hoc updates can cause performance problems since potentially large video files need to be downloaded into the
+# workspace to be able to extract the image.
+# The thumbnail preview mode affects how Opencast does ad hoc thumbnail preview image updates:
+#
+# always:  The thumbnail preview image is updated whenever required (inefficient for large video files)
+# optimized: The thumbnail preview image is updated whenever required (efficient, see notes below).
+# partial: The thumbnail preview image is only updated for uploaded thumbnails (efficient)
+#
+# Notes on "optimzied":
+# The optimized mode requires ffmpeg to be installed on the Admin node and to be included in the search path.
```

Comment:
If we're requiring the `episode.rootdir` key we should be using the composer ffmpeg key, no?